hrui/CRM/HRUI/Upgrader/Steps/4706.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+trait CRM_HRUI_Upgrader_Steps_4706 {
+
+  /**
+   * Adds "is applying for NI/SSN" field
+   */
+  public function upgrade_4706() {
+    $this->up4706_create_ssn_applying_field();
+    $this->up4706_create_activity_type();
+
+    return TRUE;
+  }
+
+  /**
+   * @throws Exception
+   */
+  private function up4706_create_ssn_applying_field() {
+    $groupName = 'Inline_Custom_Data';
+    $customGroup = civicrm_api3('CustomGroup', 'get', ['name' => $groupName]);
+
+    if ($customGroup['count'] != 1) {
+      throw new \Exception("Cannot continue without Inline_Custom_Data group");
+    }
+
+    $customGroup = array_shift($customGroup['values']);
+
+    $fieldName = 'has_applied_for_identification';
+    $customField = civicrm_api3('CustomField', 'get', ['name' => $fieldName]);
+
+    if ($customField['count'] > 0) {
+      return;
+    }
+
+    // Add "is applying" field
```

Comment:
I see the comment here indicating that the field to be added will be in present continuous tense, however it is in past tense. i.e has_applied_for_identification . Also saw on the PR overview that a new field is added which shows that contact is in process of applying for SSN.
Is this right?

Reply:
I'm open to suggestions for the field name. I found it a bit difficult to decide on using either something like "is applying for NI/SS" or "applied for NI/SSN", since using "applied" or "has applied" could mean the application was years ago. The field is probably a flag to say the user is in the process of applying and has not received their NI/SSN number yet, I could update both the comment here and the label to use something like "NI/SSN application in progress", what do you think?

Reply:
That would be fine.